tutorials/algorithms/11_trotterQRTE.ipynb

```diff
@@ -0,0 +1,1022 @@
+      "text/plain": [
+       "<Figure size 640x480 with 3 Axes>"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "axes[0].plot(exact_times, exact_energy, c='k', ls=':', label='Exact')\n",
+    "axes[1].plot(exact_times, exact_magnetization, c='k', ls=':', label='Exact')\n",
+    "axes[2].plot(exact_times, exact_correlation, c='k', ls=':', label='Exact')\n",
+    "# Select the labels of only the first axis\n",
+    "legend = fig.legend(*axes[0].get_legend_handles_labels(), bbox_to_anchor=(1., .5), loc='center left', framealpha=.5)\n",
+    "fig.tight_layout()\n",
+    "fig"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0550c4c6-9744-4ab3-a2f4-dda166c29855",
+   "metadata": {},
+   "source": [
+    "We see that, as an approximation, a Pauli-Trotter evolution isn't too far from the exact solution, but it the accuracy is limited. Let's see how to find higher order formulas to address this."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cf51369f-409e-4108-945e-cc985ec46974",
+   "metadata": {},
+   "source": [
+    "### Product formula overview\n",
+    "\n",
+    "If it isn't specified, the default product formula that `TrotterQRTE` uses is the [Lie product formula](https://en.wikipedia.org/wiki/Lie_product_formula), which is at first order."
```

Comment:
I think it would be good to clarify formally what first order is. 
Please use an actual scientific reference instead of Wikipedia.